src/database/database.module.ts

```diff
@@ -0,0 +1,27 @@
+import { config } from 'dotenv';
+config();
+
+import {
+  TypeOrmModuleAsyncOptions,
+  TypeOrmModuleOptions,
+} from '@nestjs/typeorm';
+
+
+const dbConfig: TypeOrmModuleOptions = {
+  type: 'postgres',
+  host: 'localhost',
+  port: 5432,
+  username: 'postgres',
```

Comment:
So you decided to hardcode it? :see_no_evil:

Reply:
forgot to change it back but I just did , I had to know hhh